### ISSUE TITLE
Running specific controller specs (specifying line number) causes an error

### ISSUE THREAD

**Issue by tferrerITBA** [Wolox/fictium#20]
**Describe the bug**
Specifying a specific line when running controller specs causes an error to occur, except when that specific line is the line that describes the controller. The error does not occur when no line number is specified.

**To Reproduce**
run `bundle exec rspec path/to/my/controller/spec.rb:<specific_line>`
(where `specific_line` is a line number that points to a line other than the line that contains `describe MyController do`)

**Expected behavior**
The spec should successfully run.

**Actual behavior**
Specs seem to run fine but an error occurs after running them.
```
Api::V1::UsersController
  #show
    without errors
      responds with the correct information and format
      responds with unauthorized status
      with documentation
        documents the example
    when the user is logged in
      responds with ok status
      responds with the correct information and format
      with documentation
        documents the example

An error occurred in an `after(:suite)` hook.
Failure/Error: path_object[action.method.to_sym] = create_operation(action)

NoMethodError:
  undefined method `to_sym' for nil:NilClass
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter/path_formatter.rb:7:in `add_path'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter/path_generator.rb:28:in `generate_from_action'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter/path_generator.rb:23:in `block in generate_from_resource'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter/path_generator.rb:22:in `each'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter/path_generator.rb:22:in `generate_from_resource'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter/path_generator.rb:14:in `block (2 levels) in generate'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter/path_generator.rb:13:in `each'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter/path_generator.rb:13:in `block in generate'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter/path_generator.rb:12:in `tap'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter/path_generator.rb:12:in `generate'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter.rb:52:in `create_paths'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/exporters/open_api/v3_exporter.rb:20:in `export'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/poros/document.rb:15:in `block in export'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/poros/document.rb:14:in `each'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/poros/document.rb:14:in `export'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/fictium-0.3.0/lib/fictium/rspec.rb:37:in `block (2 levels) in <top (required)>'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/example.rb:450:in `instance_exec'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/example.rb:450:in `instance_exec'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/hooks.rb:363:in `run'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/configuration.rb:2108:in `block in run_suite_hooks'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/configuration.rb:2106:in `each'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/configuration.rb:2106:in `run_suite_hooks'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/configuration.rb:2033:in `with_suite_hooks'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:116:in `block in run_specs'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/reporter.rb:74:in `report'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:115:in `run_specs'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:89:in `run'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:71:in `run'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib/rspec/core/runner.rb:45:in `invoke'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/exe/rspec:4:in `<top (required)>'
# /home/tferrer/.rbenv/versions/2.6.5/bin/rspec:23:in `load'
# /home/tferrer/.rbenv/versions/2.6.5/bin/rspec:23:in `<top (required)>'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:74:in `load'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:74:in `kernel_load'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:28:in `run'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli.rb:465:in `exec'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli.rb:27:in `dispatch'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/cli.rb:18:in `start'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/exe/bundle:30:in `block in <top (required)>'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/lib/bundler/friendly_errors.rb:124:in `with_friendly_errors'
# /home/tferrer/.rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/bundler-2.0.2/exe/bundle:22:in `<top (required)>'
# /home/tferrer/.rbenv/versions/2.6.5/bin/bundle:23:in `load'
# /home/tferrer/.rbenv/versions/2.6.5/bin/bundle:23:in `<main>'
# 
#   Showing full backtrace because every line was filtered out.
#   See docs for RSpec::Configuration#backtrace_exclusion_patterns and
#   RSpec::Configuration#backtrace_inclusion_patterns for more information.

Finished in 1.41 seconds (files took 6.81 seconds to load)
6 examples, 0 failures, 1 error occurred outside of examples
```

**Screenshots**
![image](https://user-images.githubusercontent.com/26530657/70638678-137d6800-1c18-11ea-85ff-0527236e67bd.png)

**Rails version (please complete the following information):**
6.0.1
**Ruby version (please complete the following information):**
2.6.5
**Fictium version (please complete the following information):**
0.3.0

**Additional context**
It has been tried with different controller specs but it always seems to happen. No error occurs when no line number is given (or when the line number points to the first `describe`)

**Comment by holywyvern** [comment]
It's being worked on GH-21.

**Comment by holywyvern** [comment]
Solved since `v0.4.1`, please, use that version from now on. There are yet, no breaking changes from `v0.3.X` to `v0.4.X` .
